src/@utils/codemirror.ts

```diff
@@ -0,0 +1,36 @@
+import { createTheme } from '@uiw/codemirror-themes'
+import { json } from '@codemirror/lang-json'
+
+export function checkJson(text: string) {
+  try {
+    JSON.parse(text)
+    return true
+  } catch (error) {
+    return false
+  }
+}
+
+export const oceanTheme = (marketTheme: any, field) => {
```

Comment:
we know `marketTheme` is either `light` or `dark`, which also matches `Theme` typing from `@uiw/codemirror-themes` (sadly they don't export it). So no `any` please.